### PR TITLE
feat: upgrade ui kit

### PR DESCRIPTION
**Pull request by nuanyang233**
- feat: adjust ui kit
- fix: luckdrop erc20 ui

## Description

Closes MF-1733

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

File: packages/mask/src/plugins/Collectible/SNSAdaptor/OpenSea/ListingByPriceCard.tsx [hunk 1]
```diff
@@ -146,78 +151,77 @@ export function ListingByPriceCard(props: ListingByPriceCardProps) {
...
-                        InputLabelProps={{
-                            shrink: true,
-                        }}
-                    />
+                    <>
+                        <Typography>{t('plugin_collectible_buyer_address')}</Typography>
+                        <InputBase
+                            fullWidth
+                            value={buyerAddress}
+                            // label={t('plugin_collectible_buyer_address')}
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Collectible/SNSAdaptor/OpenSea/ListingByPriceCard.tsx]: Remove commented code.

File: packages/mask/src/plugins/CryptoartAI/SNSAdaptor/MakeOfferDialog.tsx [hunk 2]
```diff
@@ -192,7 +192,7 @@ export function MakeOfferDialog(props: MakeOfferDialogProps) {
                             disableNativeToken={!paymentTokens.some((x) => isNativeTokenAddress(x.address))}
                             onAmountChange={setAmount}
                             onTokenChange={(x) => setAddress(x.address)}
-                            TokenAmountPanelProps={{
+                            ERC20InputProps={{
```
Comment (guanbinrui) [inline packages/mask/src/plugins/CryptoartAI/SNSAdaptor/MakeOfferDialog.tsx]: ERC20 is a chain specific name. Please avoid using it in a shared component.

File: packages/mask/src/plugins/Collectible/SNSAdaptor/OpenSea/ListingByPriceCard.tsx [hunk 3]
```diff
@@ -146,78 +151,77 @@ export function ListingByPriceCard(props: ListingByPriceCardProps) {
...
-                                : t('plugin_collectible_ending_price_tip'),
-                        },
                     }}
                     FungibleTokenListProps={{
                         selectedTokens: token.value ? [token.value.address] : [],
                         tokens: paymentTokens,
                         whitelist: paymentTokens.map((x) => x.address),
                     }}
                 />
+                <Typography className={classes.helperText}>
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Collectible/SNSAdaptor/OpenSea/ListingByPriceCard.tsx:165]: Can we make the helper text one of the properties of `<SelectTokenAmountPanel />`?

File: packages/mask/src/plugins/Gitcoin/SNSAdaptor/DonateDialog.tsx [hunk 4]
```diff
@@ -150,18 +149,13 @@ export function DonateDialog(props: DonateDialogProps) {
...
                     <form className={classes.form} noValidate autoComplete="off">
-                        <TokenAmountPanel
+                        <FungibleTokenInput
                             label="Amount"
                             amount={rawAmount}
                             balance={tokenBalance.value ?? '0'}
                             token={token}
                             onAmountChange={setRawAmount}
-                            SelectTokenChip={{
-                                loading: tokenBalance.loading,
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Gitcoin/SNSAdaptor/DonateDialog.tsx:160]: We lost the loading state here. It's not a equivalent replacement.

File: packages/mask/src/plugins/ITO/SNSAdaptor/SwapDialog.tsx [hunk 5]
```diff
@@ -239,7 +227,8 @@ export function SwapDialog(props: SwapDialogProps) {
                 <span className={classes.exchangeAmountText}>{formatBalance(tokenAmount, token.decimals)}</span>{' '}
                 {token.symbol}.
             </Typography>
-            <TokenAmountPanel
+            <FungibleTokenInput
+                label="Amount"
```
Comment (guanbinrui) [inline packages/mask/src/plugins/ITO/SNSAdaptor/SwapDialog.tsx]: i18n

**Comment by guanbinrui** [comment]
Please add components in the https://storybook.mask.io (Please ignore if added).

+ [ ] ActionButton
+ [ ] Switch
+ [ ] LoadingBase
+ [ ] InputBase
+ [ ] FungibleTokenInput
+ [ ] SelectTokenAmountPanel

File: packages/mask/src/plugins/Collectible/SNSAdaptor/OpenSea/ListingByPriceCard.tsx (hunk 3 above)
Comment (nuanyang233) [reply packages/mask/src/plugins/Collectible/SNSAdaptor/OpenSea/ListingByPriceCard.tsx:165]: We have used `<InputBase />` as the input component. It is already an atom level component. But, we can provide the `helperText` component.

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**